guides/micronaut-kafka/metadata.json

```diff
@@ -5,6 +5,8 @@
   "tags": ["kafka", "messaging"],
   "categories": ["Messaging"],
   "publicationDate": "2021-07-13",
+  "skipMavenTests": true,
+  "skipGradleTests": true,
```

Comment:
This passes locally for me

Reply:
The Kafka tests pass for me on Java 11, but fail on Java 17.

Reply:
How are you testing it?

Running `./gradlew micronautKafkaBuild` it passes locally for me with both:

- Java 11 https://ge.micronaut.io/s/ehkkojjwayuue#infrastructure
- Java 17 https://ge.micronaut.io/s/cpsuba3ar3keo#infrastructure

Reply:
@timyates I am running the same task as you. I'll double check it again.